### PR TITLE
Documenting the new way to inspect fastboot

### PR DESCRIPTION
**Pull request by mansona**
This PR fixes https://github.com/ember-fastboot/ember-cli-fastboot/issues/428

I decided to remove a lot of the nuance in the documentation (and refer to a previous version) because it was already quite complex with different versions and methods. That in conjunction with the fact that the Right Way™️ of doing it is the new `node --inspect` method.

Please let me know if you would prefer to include instructions for versions before v8.4 in the main documentation and I will take a crack at it. Also feel free to take as much of this as you would like and use it for a new PR 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -479,70 +479,44 @@ PRs adding or improving logging facilities are very welcome.
...
-
-Once the debug server is running, you'll want to start up the FastBoot
-server with Node in debug mode. One thing about debug mode: it makes
-everything much slower.
+First let's start up the FastBoot server with Node in debug mode. One thing
+about debug mode: it makes everything much slower.
 
 ```sh
-ember build && node --debug-brk ./node_modules/.bin/ember serve
+ember build && node --debug-brk --inspect ./node_modules/.bin/ember serve
```
Comment (rwjblue) [inline README.md]: I'm not sure exactly why we would want to `ember build &&` here. Lets remove that for now (we can bring it back if someone remembers why it was there and its still important)...

File: README.md [hunk 2]
```diff
@@ -479,70 +479,44 @@ PRs adding or improving logging facilities are very welcome.
...
 ```
 
 This does a full rebuild and then starts the FastBoot server in debug
 mode. Note that the `--debug-brk` flag will cause your app to start
```
Comment (rwjblue) [inline README.md]: `--debug-brk` isn't used, should this be updated to say `--inspect`?

File: README.md [hunk 3]
```diff
@@ -479,70 +479,44 @@ PRs adding or improving logging facilities are very welcome.
...
+#### Note Regarding Node Versions
 
-[Node 6.3](https://nodejs.org/en/blog/release/v6.3.0/) is the first version to
-offer support for the `--inspect` flag. This is intended to replace usage of
-tools like node-inspector.  However the `--inspect` flag isn't suitable for
-debugging FastBoot apps because Node does not support debugging of code in a vm
-module context (see [this issue](https://github.com/nodejs/node/issues/7593)),
-which is a module that FastBoot utilizes.
+The above method only started working for the v8.x track of Node after version v8.4.0,
+which is a fix to [this issue](https://github.com/nodejs/node/issues/7593) was merged and
```
Comment (rwjblue) [inline README.md]: Possible rewording:

> which has a fix to

File: README.md (hunk 2 above)
Comment (mansona) [inline README.md]: so `--debug-brk` does as the documentation describes and is quite useful for issues that occur *during the setup* of fastboot i.e. before you get a chance to make a single request. 

It essentially pauses the execution on the *very first node line* that is run. 👍

**Comment by mansona** [comment]
@rwjblue I have incorporated 2 of your 3 comments, I can add a section that states that `--debug-brk` is optional but useful if you like?

File: README.md (hunk 2 above)
Comment (rwjblue) [inline README.md]: Right, I know what it does, just saying that instructions haven't mentioned it to this point (the old docs said to use `--debug-brk` before you updated to specify `--inspect`).

Also, it was my understanding that using  `--inspect` and `--debug-brk` together is deprecated in recent 8.x versions (in favor `--inspect-brk`).

**Comment by rwjblue** [comment]
Thank you @mansona!

**Pull request merged**